src/overlays/actors/ovl_Bg_Ladder/z_bg_ladder.c

```diff
@@ -19,22 +37,129 @@ const ActorInit Bg_Ladder_InitVars = {
     (ActorFunc)BgLadder_Init,
     (ActorFunc)BgLadder_Destroy,
     (ActorFunc)BgLadder_Update,
-    (ActorFunc)BgLadder_Draw
+    (ActorFunc)BgLadder_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Bg_Ladder_0x80A4D9F0/BgLadder_Init.asm")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+static u32 sLadderDLists[] = {
```

Comment:
Looks like a list of Gfx not u32

That's all I saw, the draw function looks weird to me but I think that's just me.

Looks good!